base/generation_lock.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include "base/generation_lock.hpp"
 
+#include <functional>
```

Comment:
could you make the `include` in alphabetical order?

Reply:
Done.